src/arrayMethodSplice.js

```diff
@@ -5,7 +5,78 @@
  */
 function applyCustomSplice() {
   [].__proto__.splice2 = function(start, deleteCount, ...items) {
-    // write code here
+    const deleteCountSplice = deleteCount;
```

Comment:
I know it is hard to add names to variables, but `deleteCountSplice` - not a good name

Reply:
create let and mutate it later or just use `deleteCount` below